### PR TITLE
Add distribution and budget params to api-server

### PR DESCRIPTION
**Pull request by 0xbala-k**

### CODE REVIEW

File: api/chains/chains.go [hunk 1]
```diff
@@ -1269,3 +1269,87 @@ func GetChainsStatuses(db *database.Database) gin.HandlerFunc {
 		c.JSON(http.StatusOK, res)
 	}
 }
+
+// GetDistributionParams returns the distribution params of a specific chain
+// @Summary Gets the ditribution params of a chain
+// @Description Gets distribution params
```
Comment (sgerogia) [inline api/chains/chains.go]: Maybe link to Cosmos docs on the concept?

File: api/chains/chains.go [hunk 2]
```diff
@@ -1269,3 +1269,87 @@ func GetChainsStatuses(db *database.Database) gin.HandlerFunc {
...
+			return
+		}
+
+		c.Data(http.StatusOK, gin.MIMEJSON, sdkRes.DistributionParams)
+	}
+}
+
+// GetBudgetParams returns the budget params of a specific chain
+// @Summary Gets the budget params of a chain
+// @Description Gets budget params
```
Comment (sgerogia) [inline api/chains/chains.go]: Maybe link to Cosmos docs on the concept?

**Review by sgerogia — APPROVED** [review]

File: api/chains/chains.go [hunk 3]
```diff
@@ -1269,3 +1269,87 @@ func GetChainsStatuses(db *database.Database) gin.HandlerFunc {
...
+				fmt.Errorf("cannot retrieve distribution params from sdk-service: %w", err),
+				"name",
+				chain.ChainName,
+			)
+			_ = c.Error(e)
+
+			return
+		}
+
+		c.Data(http.StatusOK, gin.MIMEJSON, sdkRes.DistributionParams)
```
Comment (pitasi) [inline api/chains/chains.go:1311]: any reason we don't do like the method above?

```suggestion
		c.JSON(http.StatusOK, sdkRes.DistributionParams)
```

File: api/chains/chains.go [hunk 4]
```diff
@@ -1269,3 +1269,87 @@ func GetChainsStatuses(db *database.Database) gin.HandlerFunc {
...
+				fmt.Errorf("cannot retrieve budget params from sdk-service: %w", err),
+				"name",
+				chain.ChainName,
+			)
+			_ = c.Error(e)
+
+			return
+		}
+
+		c.Data(http.StatusOK, gin.MIMEJSON, sdkRes.BudgetParams)
```
Comment (pitasi) [inline api/chains/chains.go:1353]: any reason we don't do like the method above?

```suggestion
		c.JSON(http.StatusOK, sdkRes.BudgetParams)
```

Comment (0xbala-k) [reply api/chains/chains.go:1353]: No particular reason. Both give the same result. It was from copy pasted code, so didn't change it.

File: api/chains/chains.go (hunk 3 above)
Comment (0xbala-k) [reply api/chains/chains.go:1311]: No particular reason. Both give the same result. It was from copy pasted code, so didn't change it.

**Review by DeshErBojhaa — APPROVED** [review]
🎉

**Pull request merged**